common/meta/raft-store/src/config.rs

```diff
@@ -144,8 +147,16 @@ impl RaftConfig {
         <Self as Parser>::parse_from(&Vec::<&'static str>::new())
     }
 
-    pub fn raft_api_addr(&self) -> String {
-        format!("{}:{}", self.raft_api_host, self.raft_api_port)
+    /// Support ip address and hostname
+    pub async fn raft_api_addr(&self) -> Result<String> {
+        if IPAddress::is_valid(self.raft_api_host.as_str()) {
```

Comment:
How about using `std::net::IpAddr` so that we don't need to depend on `ipaddress`?

Reply:
> How about using `std::net::IpAddr` so that we don't need to depend on `ipaddress`?

That makes sense, it will cleaner.